parent/pom.xml

```diff
@@ -241,11 +241,12 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
```

Comment:
I'm a fan of spelling this include/exclude out explicitly for two reasons:

a) to make it easier for downstream devs to understand why their class is/isn't running as expected
b) to make it easier for downstream devs to override if they need to

Sure, they can google into https://eclipse.org/tycho/sitedocs/tycho-surefire/tycho-surefire-plugin/test-mojo.html#includes but if we can be explicit in the parent pom, isn't that better/easier/one-stop-shopping?